Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+// vim: ft=groovy
+
+properties([
+    parameters([
+        string(name: 'TARGET', defaultValue: '', description: '')
```

Comment:
I think the default value should be `all`, otherwise all pushes to this repo will end up in failed builds (that take a long while to fail, too).

Reply:
Also, why is the `description` blank?